src/network_models/copperplate_model.jl

```diff
@@ -1,19 +1,28 @@
-function copper_plate(
-    optimization_container::OptimizationContainer,
-    expression::Symbol,
-    bus_count::Int,
-)
+function copper_plate!(::Type{CopperPlatePowerModel}, optimization_container::OptimizationContainer)
     time_steps = model_time_steps(optimization_container)
-    remove_undef!(optimization_container.expressions[expression])
-
     constraint_val = JuMPConstraintArray(undef, time_steps)
     assign_constraint!(optimization_container, "CopperPlateBalance", constraint_val)
+    expressions = get_expression(optimization_container, :system_balance_active)
+    remove_undef!(expressions)
+    jump_model = get_jump_model(optimization_container)
 
     for t in time_steps
-        constraint_val[t] = JuMP.@constraint(
-            optimization_container.JuMPmodel,
-            sum(optimization_container.expressions[expression].data[1:bus_count, t]) == 0
-        )
+        constraint_val[t] = JuMP.@constraint(jump_model, expressions[t] == 0)
```

Comment:
@claytonpbarrows this implementation reduces the time to build the copperplate from 20 seconds to 400 ms.